### PR TITLE
Basic refactor of agile section

### PR DESCRIPTION
**Pull request by TessFerrandez**
What is done

- Cleaning up structure, to flatten the folder structure
- Fixing links
- Moving content around to where it fits better to avoid readers having to look at multiple docs to get the data

What is not done

Full overview of the content, this is merely a first standardization refactoring to delete superfluous pages and tables of content as the GitHub pages already has nice functionality for automatic TOCs and navigation

### CODE REVIEW

File: docs/agile-development/advanced-topics/team-agreements/working-agreements.md [hunk 1]
```diff
@@ -37,21 +37,21 @@ their own, and adjust times, communication channels, branch naming policies etc.
 
 ## Scrum Rhythm
 
-| Activity                                              | When                  | Duration | Who          | Accountable  | Goal                                                                       |
```
Comment (Doommius) [inline docs/agile-development/advanced-topics/team-agreements/working-agreements.md:40]: nitpick: i personally prefer the expanded tables as it's easier to work with if txt mode if the entire table is formatted with spaces/tabs.

**Review by Doommius — APPROVED** [review]
Looks good! I really like the more compacted view instead of having many small pages with very little information.

File: docs/agile-development/advanced-topics/team-agreements/working-agreements.md (hunk 1 above)
Comment (TessFerrandez) [reply docs/agile-development/advanced-topics/team-agreements/working-agreements.md:40]: ok, thank you for the input

**Review by nyouens — APPROVED** [review]
This is a great contribution Tess, thank you!

**Pull request merged**